### PR TITLE
Disable hidden timestamp attribute by default

### PR DESCRIPTION
**Pull request by ethho**
Adds configuration flag `datajoint.config["add_hidden_timestamp"]`, set to `False` by default, which toggles automatic addition of the timestamp hidden attribute. Patches #1091, #1180.

### CODE REVIEW

**Comment by ethho** [comment]
CI runs blocked by either https://github.com/datajoint/nginx-docker/pull/56 or #1183.

File: datajoint/declare.py [hunk 1]
```diff
@@ -311,17 +312,18 @@ def declare(full_table_name, definition, context):
...
 
-    metadata_attr_sql = [
-        "`_{full_table_name}_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP"
-    ]
-    attribute_sql.extend(
-        attr.format(
-            full_table_name=sha1(
-                full_table_name.replace("`", "").encode("utf-8")
-            ).hexdigest()
+    if config.get("enable_hidden_attributes", False):
```
Comment (dimitri-yatsenko) [inline datajoint/declare.py:315]: How about changing to `add_hidden_timestamp` to make this feature more specific. Hidden attributes will always be enabled. But the default timestamp will be configurable.

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]
make the setting name more specific to implicit timestamps.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
 ## Release notes
 
 ### 0.14.3 -- Sep 20, 2024
```
Comment (dimitri-yatsenko) [inline CHANGELOG.md:3]: Sep 23 perhaps?

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**